arch/arc/core/reset.S

```diff
@@ -121,6 +121,35 @@ invalidate_dcache:
 
 done_cache_invalidate:
 
+/*
+ * ARC Custom Initialization
+ * Force Reset MPU (v3) registers
+ * For some test scenarios, this is needed to apply additional configurations
+ */
+#if defined(CONFIG_ARC_CUSTOM_INIT)
+#if CONFIG_ARC_MPU_VER == 3
+	/* Set default reset value to _ARC_V2_MPU_EN */
+	mov_s r1, 0x400181C0
```

Comment:
Please don't use literals like that. Create a definition right here with meaningful name & comments.